sdk/appplatform/Microsoft.Azure.Management.AppPlatform/src/Microsoft.Azure.Management.AppPlatform.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.AppPlatform</PackageId>
+    <Description>Provides developers with libraries for the Azure Spring Cloud under Azure Resource manager to deploy, start, stop and scale Spring Cloud applications.</Description>
+    <Version>1.0.0</Version>
```

Comment:
If your sdk is going to be preview the version should be 0.1.0-preview. If you decide it's stable enough then it's fine

Reply:
updated version to 0.1.0-preview